service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/support/MigrateCaseController.java

```diff
@@ -42,19 +36,23 @@
 @Slf4j
 public class MigrateCaseController extends CallbackController {
     private static final String MIGRATION_ID_KEY = "migrationId";
+    private static final String PLACEMENT = "placements";
+    private static final String PLACEMENT_NON_CONFIDENTIAL = "placementsNonConfidential";
+    private static final String PLACEMENT_NON_CONFIDENTIAL_NOTICES = "placementsNonConfidentialNotices";
 
     private final MigrateCaseService migrateCaseService;
     private final DocumentListService documentListService;
     private final ManageOrderDocumentScopedFieldsCalculator fieldsCalculator;
 
     private final Map<String, Consumer<CaseDetails>> migrations = Map.of(
-        "DFPL-1144", this::run1144,
-        "DFPL-872rollback", this::run872Rollback,
-        "DFPL-1072", this::run1072,
-        "DFPL-1163", this::run1163,
-        "DFPL-1165", this::run1165,
-        "DFPL-1192", this::run1192,
-        "DFPL-1215", this::run1215
+        "DFPL-1204", this::run1204,
+        "DFPL-1202", this::run1202,
+        "DFPL-1195", this::run1195,
+        "DFPL-1065", this::run1065,
+        "DFPL-1029", this::run1029,
+        "DFPL-1162", this::run1162,
```

Comment:
1029 & 1162 can be removed.

Reply:
Think you can just simply put your PRs
`"DFPL-1202", this::run1202,
"DFPL-1195", this::run1195,
"DFPL-1204", this::run1204,
"DFPL-1218", this::run1218`